sql/npc_list.sql

```diff
@@ -27359,14 +27359,14 @@ INSERT INTO `npc_list` VALUES (17740016,'Eggsperiment_B','Eggsperiment B',127,-1
 INSERT INTO `npc_list` VALUES (17740017,'Eggsperiment_I','Eggsperiment I',127,-155.000,-4.000,-27.000,0,40,40,0,1,40,6,2075,0x01001E0100107120173017401750006000700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17740018,'Eggsperiment_T','Eggsperiment T',127,-155.000,-4.000,-33.000,0,40,40,0,1,40,6,2075,0x01001E0282117120173017401750006000700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17740019,'Moogle','Moogle',15,-260.221,-12.021,-77.689,1,40,40,0,0,0,2,3,0x0000520000000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17740020,'Moogle','Moogle',40,-259.440,-12.021,-79.538,1,40,40,0,0,0,2,4194307,0x0000520000000000000000000000000000000000,0,NULL,1);
--- INSERT INTO `npc_list` VALUES (17740021,'blank','_',0,0,0,0,0,50,50,0,0,112,2,2051,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17740022,'Moogle','Moogle',105,-293.121,-12.021,-76.843,1,50,50,0,0,0,2,3,0x0000390900000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17740023,'Chacharoon','Chacharoon',152,-293.500,-12.020,-59.500,1,50,50,0,0,0,2,27,0x00005F0B00000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17740020,'History_Moogle','Moogle',40,-259.440,-12.021,-79.538,1,40,40,0,0,0,0,4194307,0x0000520000000000000000000000000000000000,0,NULL,1);
```

Comment:
Can you confirm that this Moogle is _only_ used for this event? Else we're going to have potential conflicts in the future.  This might be something to just leave in the db for a record, and create dynamically

Reply:
@claywar I can already tell you with 100% certainty that it WILL be used for multiple things. SE reuses them as well as shuffling them in and out. This is why I was pushing for dynamic entity moogles on the Easter even PR.

Same thing here, using a DE and adding in a `facePlayer()` before starting the event will make them look just like retail despite the NPC ID not matching.

Reply:
Currently, I can't confirm or deny that this particular Moogle is unused by other events. It differs from, for example, the ID(s) being used by the Eggstravaganza Moogles. I am happy to monitor as seasonal events rotate in/out to check on it, however.

If there are other events discovered where it is evident they are using this same ID, could approach it by renaming the moogle based on the sequence they appear in the Zone (e.g. `Moogle_01`), add conditionals to the script, and add additional "disappear" handling.

Reply:
> Currently, I can't confirm or deny that this particular Moogle is unused by other events. It differs from, for example, the ID(s) being used by the Eggstravaganza Moogles. I am happy to monitor as seasonal events rotate in/out to check on it, however.
> 
> If there are other events discovered where it is evident they are using this same ID, could approach it by renaming the moogle based on the sequence they appear in the Zone (e.g. `Moogle_01`), add conditionals to the script, and add additional "disappear" handling.

They shift these around and even change the total number of moogles regularly, but what they use the 1st/2nd/3rd/4th moogle for changes while still occupying the same relative position. **This is known** and I've watched those moogles shift during 6+ years of npc id updating and seeing mogs captured during events repeatedly. 

So we either start using DEs or consign ppl to never being able to enable these out of season and having to deal with the ID shifting and needing to make sure its updated (client update time) immediately  ***forever*** so people can actually start the event when SE does. Starting a pattern of just using a DE eliminates all of that maintenance cost and "just works".

Reply:
Firstly, I appreciate all the feedback on the draft.  Secondly, regarding using the actual NPC ID vs a DE:

Whichever of these two methods is the stated preference by the project is the one I'll work on implementing. If the preference is Dynamic Entries, should this apply to all of the actors (Vanaversary Chest, Vanaversary Moogle, History Chest, Chacharoon, etc), or is the concern about the specific Moogle that has been observed to have been reused or ID Shifted?

Reply:
its even more important for non moogles. in theory if we insisted on never using DEs we could at least have logic in a re-used moogle check which role he's supposed to be playing. But Chacharoon doesn't have that option when SE rips him out of the dat in a few months time.

I've just asked the rest of the team to voice any objection they have to using DE to evade all future maintenance cost - there are pretty much zero drawbacks to it other than one I already created a workaround for. We really need to get a good standard going on these things early, because when the next person decides to add a missing seasonal event they'll be looking at what is done here to guide them.

Reply:
verdict: do the DE

Reply:
> verdict: do the DE

Swapped back to draft while I work out moving this from fixed to DE.

Reply:
Current draft supports Dynamic Entities with some caveats:

* Cut scenes do not properly animate Dynamic Entities (DE ID is not what the CS expects)
* Player ends up looking at an incorrect xyz due to the CS Enity ID differing from the DE.

> We really need to get a good standard going on these things early, because when the next person decides to add a missing seasonal event they'll be looking at what is done here to guide them.

Next major commit will rework back to Static Entries, but leaving the DE commit in place as a potential example.

Reply:
we have means to turn both entities already. its a separate function call. We can even perform other animations as well. (NPC waving and what have you) This is not an impassable barrier.

I coudl modify the recently added `facePlayer()` to also turn the player and save doing an extra call if this is typical behavior.